### PR TITLE
replacing perfectomobile plugin with a new plugin named: perfecto

### PR DESCRIPTION
**Pull request by genesisthomas**
Please approve this PR as perfecto mobile plugin is deprecated and not maintained.

### CODE REVIEW

**Comment by daniel-beck** [comment]
Please also define a deprecation and link to some documentation (can be the old GitHub repo whose readme explains it), see https://github.com/jenkins-infra/update-center2#deprecations

**Comment by genesisthomas** [comment]
dear @daniel-beck , We don't have access to the perfectomobile Jenkins plugin and hence we couldn't add the deprecated label. This is the reason why we raised this PR.

Old perfecto Jenkins url: https://plugins.jenkins.io/perfectomobile/
Old perfecto jenkins github: https://github.com/jenkinsci/perfectomobile-plugin
Best readme section for the old plugin: https://plugins.jenkins.io/perfectomobile/

Please let us know if we are missing something/ need to do something else.

Looking forward for your reply.

**Comment by daniel-beck** [comment]
I see, both Oleg and I provided specific actionable feedback in https://issues.jenkins-ci.org/browse/HOSTING-1004 which you seemed to ignore.

Still, it makes no sense to stop publishing plugins without an explanation to inform current users about it, so please add that somewhere and add it to the deprecations list.

**Comment by genesisthomas** [comment]
Dear @daniel-beck , thanks for your comment. I have updated the deprecations list. The url (https://developers.perfectomobile.com/display/PD/Legacy+%7C+Jenkins+plugin) will be made available to public view shortly. Attaching screenshots for your quick reference.

We were prioritising to first complete the development of new plugin and then follow up with you on deprecating the old one. Thank you guys for being of great help to us during this transition.
Screenshot 2020-08-11 at 9 44 35 PM

File: resources/label-definitions.properties [hunk 1]
```diff
@@ -708,7 +708,8 @@ pegdown-formatter=ui
 pending-changes=scm-related
 people-redirector=external
 percentage-du-node-column=listview-column
-perfectomobile=external ios android
+perfectomobile=deprecated
+perfecto=ios android tunnel external
```
Comment (daniel-beck) [inline resources/label-definitions.properties:712]:
```suggestion
perfecto=ios android external
```

`tunnel` is new and also not in https://github.com/jenkins-infra/update-center2/blob/master/resources/allowed-github-topics.properties

Let's keep that conversation separate from the multiple things that this PR is already doing.

**Comment by daniel-beck** [comment]
@genesisthomas That looks reasonable.

I plan to merge this PR (with my suggested change applied) once the linked page exists.

**Comment by genesisthomas** [comment]
Dear @daniel-beck , Really appreciate your suggestions & I have pushed them as well. I'm also following up internally to enable the linked page to be made available publicly. Thank you so much for your quick responses on our PR.

**Comment by genesisthomas** [comment]
dear @daniel-beck , the pages are visible now. Kindly accept this PR

**Comment by daniel-beck** [comment]
@genesisthomas Thanks, we'll merge this later today.

**Comment by daniel-beck** [comment]
Sorry for the delay, yesterday's security update publishing took a lot longer than expected.

**Comment by genesisthomas** [comment]
Dear @daniel-beck , I'm working on the security updates as we speak. You guys have been amazing to us. All is well.

**Pull request merged**